Core/include/Acts/Seeding/SeedFilter.ipp

```diff
@@ -72,7 +73,7 @@ void SeedFilter<external_spacepoint_t>::filterSeeds_2SpFixed(
 
   size_t beginCompTopIndex = 0;
   // loop over top SPs and other compatible top SP candidates
-  for (auto& topSPIndex : topSPIndexVec) {
+  for (const std::size_t& topSPIndex : topSPIndexVec) {
```

Comment:
Does that even make sense? `size_t` is the same size as a pointer, so maybe just take a copy?